app/apollo/resolvers/group.js

```diff
@@ -465,6 +480,9 @@ const groupResolvers = {
         */
         groupsRbacSync( groups, { resync: false }, context ).catch(function(){/*ignore*/});
 
+        // Allow graphQL plugins to retrieve more information
+        context.pluginContext = {clusterId: clusterId, groupDetails: groupObjsToAdd};
```

Comment:
pluginContext should be like `{ name: clusterName, uuid: clusterId, groups: [{name,uuid}]`